jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -824,3 +836,10 @@ pipeline {
         }
     }
 }
+
+def markStageUnstableIfPluginsFailedToBuild() {
+    def stageLogs = getLogsForStage(stageName: "${STAGE_NAME}")
+    if (stageLogs.any{e -> e.contains('Failed plugins are')}) {
```

Comment:
Seems like we are reading through all of the logs from Jenkins stage even if there are no failed plugins right? 
I feel this might be really inefficient and delay the building process. Can we maybe modify the `--continue-on-error` from the build workflow so that it could send signal to whenever it skips one plugin?
It could be having a new variable storing a list of failed plugins or just a boolean saying we continued on error.

Reply:
The only time that string is thrown as an error is when continue on error is enabled. Otherwise it is not. I think it will be too many if/else statements to call this function if continue on error is enabled. 
Another work around is to only call this upon success as that when this error string is thrown. Will make that change. Making this PR as draft until dependent PRs are merged.

Reply:
Made the changes! The function is only called when `CONTINUE_ON_ERROR = true` and only for success stage. 
Thanks!